external/esp_idf_port/esp32/esp32_wifi_os_adapter.c

```diff
@@ -435,7 +435,7 @@ static void IRAM_ATTR timer_setfn_wrapper(void *ptimer, void *pfunction, void *p
 		}
 		etimer->timer_func = pfunction;
 		etimer->timer_expire = (uint32_t) parg;
-		etimer->timer_next = (struct work_s *)malloc(sizeof(struct work_s));
+		etimer->timer_next = (struct work_s *)zalloc(sizeof(struct work_s));
```

Comment:
agree  with this modification. below is the detail reason.

timer call sequency is 
timer_setfn_wrapper-> once_timerfn

if not init timer_next in the function timer_setfn_wrapper,
the content in the work_s structure is random, depend on the malloced memory that time.
if there is invalid worker value(not null) in it ,  then run work_cancel  int once_timerfn will cause crash.
reference:
struct work_s {
    struct dq_entry_s dq;       /* Implements a doubly linked list */
    worker_t worker;            /* Work callback */
    FAR void *arg;              /* Callback argument */
    clock_t qtime;          /* Time work queued */
    clock_t delay;          /* Delay until work performed */
};